packages/grafana-data/src/field/displayProcessor.ts

```diff
@@ -75,7 +75,7 @@ export function getDisplayProcessor(options?: DisplayProcessorOptions): DisplayP
   const formatFunc = getValueFormat(unit || 'none');
   const scaleFunc = getScaleCalculator(field, options.theme);
 
-  return (value: any) => {
+  return (value: any, decimals?: number | null) => {
```

Comment:
do we need the | null?
```suggestion
  return (value: any, decimals?: number) => {
```

Reply:
i think we need to stay in line with https://github.com/grafana/grafana/blob/277a83215c1e20ff8c93768f93edf92d60deee5f/packages/grafana-data/src/types/displayValue.ts#L38